sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -167,68 +167,123 @@ ConfigurationAsyncClient client = new ConfigurationClientBuilder()
+Retrieve a previously stored configuration setting by calling `getConfigurationSetting`.
 ```Java
-ConfigurationClient client = new ConfigurationClientBuilder()
-        .connectionString(connectionString)
-        .buildClient();
-client.setSetting("some_key", "some_label", "some_value");
-ConfigurationSetting setting = client.setSetting("some_key", "some_label", "new_value");
+ConfigurationSetting setting = client.setConfigurationSetting("some_key", "some_label", "some_value");
+ConfigurationSetting retrievedSetting = client.getConfigurationSetting("some_key", "some_label");
+```
+For conditional request,
+```Java
+Response<ConfigurationSetting> settingResponse = client.getConfigurationSettingWithResponse(setting, null, true, Context.NONE);
 ```
 
-### Delete a Configuration Setting
+### Update an existing configuration setting
 
-Delete an existing Configuration Setting by calling deleteSetting.
+Update an existing configuration setting by calling `setConfigurationSetting`.
 ```Java
-ConfigurationClient client = new ConfigurationClientBuilder()
-        .connectionString(connectionString)
-        .buildClient();
-client.setSetting("some_key", "some_label", "some_value");
-ConfigurationSetting setting = client.deleteSetting("some_key", "some_label");
+ConfigurationSetting setting = client.setConfigurationSetting("some_key", "some_label", "some_value");
+ConfigurationSetting updatedSetting = client.setConfigurationSetting("some_key", "some_label", "new_value");
+```
+For conditional request,
+```Java
+Response<ConfigurationSetting> settingResponse = client.setConfigurationSettingWithResponse(setting, true, Context.NONE);
+```
+
+### Delete a configuration setting
+
+Delete an existing configuration setting by calling `deleteConfigurationSetting`.
+```Java
+ConfigurationSetting setting = client.setConfigurationSetting("some_key", "some_label", "some_value");
+ConfigurationSetting deletedSetting = client.deleteConfigurationSetting("some_key", "some_label");
+```
+For conditional request,
+```Java
+Response<ConfigurationSetting> settingResponse = client.deleteConfigurationSettingWithResponse(setting, true, Context.NONE);
+```
+
+### List configuration settings with multiple keys
+
+List multiple configuration settings by calling `listConfigurationSettings`.
+Pass null `SettingSelector` into the method If you want to fetch all configuration settings.
+```Java
+String key = "some_key";
+String key2 = "new_key";
+client.setConfigurationSetting(key, "some_label", "some_value");
+client.setConfigurationSetting(key2, "new_label", "new_value");
+SettingSelector selector = new SettingSelector().setKeys(key, key2);
+PagedIterable<ConfigurationSetting> settings = client.listConfigurationSettings(selector);
+```
+
+### List revisions of multiple configuration settings
+
+List all revision of a configuration settings by calling `listRevisions`.
+```Java
+String key = "revisionKey";
+String Key2 = "newRevisionKey";
```

Comment:
```java
String key2 = "newRevisionKey";
```

Reply:
I don't think you need key2 in this sample. It's confusing because the name is also called "newRevisionKey"